core/src/main/java/io/undertow/server/DirectByteBufferDeallocator.java

```diff
@@ -33,10 +33,8 @@
         Unsafe tmpUnsafe = null;
         if (version < 9) {
             try {
-                tmpCleaner = Class.forName("java.nio.DirectByteBuffer").getMethod("cleaner");
-                tmpCleaner.setAccessible(true);
-                tmpCleanerClean = Class.forName("sun.misc.Cleaner").getMethod("clean");
-                tmpCleanerClean.setAccessible(true);
+                tmpCleaner = new GetAccesibleMethod("java.nio.DirectByteBuffer", "cleaner").getMethod();
```

Comment:
For consistency purposes, can this be swapped by a "private Method getAccesibleMethod(String methodName)" that is invoked like getUnsafe0 a few lines below?